lambda/marchon.py

```diff
@@ -238,10 +288,19 @@ def lambda_handler(event=None, context=None, dry_run=False):
 def events_lambda_handler(event=None, context=None, dry_run=False):
     sheet = get_event_data()
     log.info('sheet=%s\n', sheet)
+
+    action_network_events = get_action_network_events()
+    log.info('action_network_events=%s\n', action_network_events)
+
+    all_events = {**sheet, **action_network_events}
```

Comment:
this is much more cryptic and harder to read than `data.update(get_action_network_events)` where `data = get_event_data()`

Reply:
I disagree.

Reply:
I thought about this.  I agree now.

Reply:
Confession: I was excited about the python 3.5 syntax.